docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>the variable names here are confusing. It seems as if you were manipulating RGB values already, when you are only defining shapes.</p><p> </p><pre class="ql-syntax" spellcheck="false">def setup_split_rgb(outs, ins):
    for r_shape, g_shape, b_shape, in_shape in zip(outs[0], outs[1], outs[2], ins[0]):
        r_shape[:] = g_shape[:] = b_shape[:] = in_shape[0:2]

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>